### PR TITLE
Remove extra empty line from Applied Visual Design Challenge

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@thecodingaviator Go ahead and add this change for the non-English versions of this challenge as there would be nothing to be translated.  We will fix them all in one swipe

**Comment by thecodingaviator** [comment]
@RandellDawson Changes made

**Review by lawrencejon — COMMENTED** [review]
Good work. Simple fixes like this can make make quite an impact on the user experience. Keep up the good work.

**Review by RandellDawson — APPROVED** [review]
LGTM - I approve.

In a separate PR, it would not be a bad idea to go ahead an make the solution look like the rest (without the `code = ``;`  It is not affecting the ability of the test runner to validate, but it does not look as clean as just having the full code (html and css together).

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Comment by thecodingaviator** [comment]
> In a separate PR, it would not be a bad idea to go ahead an make the solution look like the rest (without the `` code = ``; `` It is not affecting the ability of the test runner to validate, but it does not look as clean as just having the full code (html and css together).

@RandellDawson I'm not sure I follow through. Where exactly are you referring to?

**Comment by RandellDawson** [comment]
@thecodingaviator I was referring to the fact that the solution section should look like:

````html
## Solution
<section id='solution'>

```html
<style>
.center
{
  position: absolute;
  margin: auto;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  width: 100px;
  height: 100px;
  background-color: transparent;
  border-radius: 50%;
  box-shadow: 25px 10px 0px 0 blue;
}

</style>
<div class="center"></div>
```

</section>
````
instead of:
````
## Solution
<section id='solution'>

```js
var code = ".center {background-color: transparent; border-radius: 50%; box-shadow: 25px 10px 0px 0 blue;}"
```

</section>
````
Basically, it should include the original challenge seed code plush the parts that need to be changed to pass the challenge.

**Comment by thecodingaviator** [comment]
@RandellDawson Got your point. The thing is, this would 99% surely would not be the only challenge that has this problem so we can go ahead and create an issue to track these changes IMO

**Comment by RandellDawson** [comment]
I think we will just deal with these as they pop up instead of creating an issue for them.
